integrations/airflow/marquez_airflow/marquez.py

```diff
@@ -15,108 +22,187 @@
 log = logging.getLogger(__name__)
 
 
-class Marquez:
+class MarquezAdapter:
```

Comment:
Some class and method level doc comments would be extremely useful here, as I can imagine others wanting to tap in here for their own purposes.